lightning/src/ln/channelmanager.rs

```diff
@@ -1277,6 +1277,14 @@ pub struct ChannelDetails {
 	pub is_usable: bool,
 	/// True if this channel is (or will be) publicly-announced.
 	pub is_public: bool,
+	/// The minimum amount of msats that can be forwarded by the channel holder in an HTLC.
+	pub holder_htlc_minimum_msat: u64,
+	/// The maximum amount of msats that can be forwarded by a channel holder in an HTLC.
+	pub holder_htlc_maximum_msat: u64,
+	/// The minimum amount of msats that can be forwarded by a channel counterparty in an HTLC.
+	pub counterparty_htlc_minimum_msat: u64,
```

Comment:
I think this and the next one probably belong in `CounterpartyForwardingInfo` instead.